src/components/LegalButton.js

```diff
@@ -0,0 +1,14 @@
+import React, { PureComponent } from 'react';
+import './LegalButton.css';
+
+export default class LegalButton extends PureComponent {
+  render() {
+    const { onClick } = this.props;
```

Comment:
pass a property color in here.
Then apply an inline style that sets the color. Thereby you can pass the desired color into the component